src/clj/game/core/cards.clj

```diff
@@ -197,6 +197,11 @@
   (let [hiveminds (filter #(= (:title %) "Hivemind") (all-installed state :runner))]
     (reduce + (map #(get-in % [:counter :virus] 0) (cons card hiveminds)))))
 
+(defn count-virus-programs
+  "Calculate the number of virus programs in play"
+  [state]
+  (count (filter #(= false (has-subtype? % "Virus")) (all-installed state :runner))))
```

Comment:
Instead of `filter` you could try `remove` and drop the `= false` comparison.

Reply:
Yeah, I realized that I had the filter predicate incorrect in my head. Needs to return a sequence of cards where the predicate `:subtype == "Virus"` is true, not the other way around.

Fixed, in an accidental force-push.